manager/Dockerfile

```diff
@@ -2,15 +2,15 @@
 #
 # VERSION               0.3
 
-FROM node as web-builder
+FROM node AS web-builder
```

Comment:
```suggestion
FROM node:22.12 AS web-builder
```

Reply:
Shouldn't rebuilding like that have picked up `node:latest`? That would have been node 23.3. I'm fine with pinning to version though, especially since 22 is an LTS release.